tests/test_freetextsearch.py

```diff
@@ -51,7 +1364,6 @@ def testSearchFields(self):
         self.assertTrue(response.status_code == 200)
         json_response = json.loads(response.data.decode('utf-8'))
         self.assertEqual(len(json_response['data']), 1)
```

Comment:
I thought we wanted to have no highlights unless it is specifically asked for in the fields?!